examples/sprite_batch/CMakeLists.txt

```diff
@@ -0,0 +1,25 @@
+
```

Comment:
Can you remove this leading empty newline? I recently removed all these empty leading newlines from all the example build scripts.